deploy/docker-compose/README.md

```diff
@@ -0,0 +1,98 @@
+# Docker Compose Deployment
+
+## Quick Start
+
+This is a simple example that shows you how to connect `nm-vllm` with production monitoring via Prometheus and Grafana via Docker Compose.
+
+## Install
+
+Make sure you have Docker and Docker Compose installed.
+- [`docker`](https://docs.docker.com/engine/install/)
+- [`docker compose`](https://docs.docker.com/compose/install/linux/#install-using-the-repository)
+
+> Note: `nm-vllm` requires a GPU. Make sure you have installed the [Nvidia Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html).
+
+## Deploy
+
+### Launch
+
+The following spins up an `nm-vllm` instance, a Prometheus server, and a Grafana server:
+
+```bash
+docker compose up
+```
```

Comment:
Maybe it's obvious enough, but I think it could be useful to have something here that makes it clear that you need to clone the repo and navigate to this directory before running this command.